etc/docker/demo-base/Dockerfile

```diff
@@ -66,7 +66,8 @@ RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && \
 ADD fix-permissions /usr/local/bin/fix-permissions
 # Create jovyan user with UID=1000 and in the 'users' group
 # and make sure these dirs are writable by the `users` group.
-RUN groupadd wheel -g 11 && \
+RUN userdel -r mambauser && \
```

Comment:
Why is this necessary?  I wonder if that's part of my build issue.

Reply:
The mamba image has created a user with 1000 groupid as the jovyan user we use.